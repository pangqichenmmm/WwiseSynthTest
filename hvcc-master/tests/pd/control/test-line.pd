#N canvas 750 181 334 378 10;
#X msg 90 148 39;
#X msg 178 147 stop;
#X msg 127 149 set 5;
#X obj 46 19 loadbang;
#X obj 46 316 print;
#X obj 128 105 t b b;
#X obj 46 41 t b b b b;
#X obj 63 117 t b b;
#X msg 241 111 1 10;
#X obj 46 294 line 0 2;
#X obj 128 83 del 5;
#X obj 72 87 del 10;
#X msg 141 248 5;
#X msg 22 149 0 5;
#X obj 14 90 del 15;
#X connect 0 0 9 0;
#X connect 1 0 9 0;
#X connect 2 0 9 0;
#X connect 3 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 4 0;
#X connect 5 1 1 0;
#X connect 6 0 14 0;
#X connect 6 1 11 0;
#X connect 6 2 10 0;
#X connect 6 3 8 0;
#X connect 7 0 12 0;
#X connect 7 1 0 0;
#X connect 8 0 9 0;
#X connect 9 0 4 0;
#X connect 10 0 5 0;
#X connect 11 0 7 0;
#X connect 12 0 9 2;
#X connect 13 0 9 0;
#X connect 14 0 13 0;
