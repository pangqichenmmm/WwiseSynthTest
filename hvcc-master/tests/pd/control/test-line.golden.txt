[@ 0.000] print: 0
[@ 2.000] print: 0.2
[@ 4.000] print: 0.4
[@ 5.000] print: bang
[@ 10.000] print: 39
[@ 15.000] print: 39
[@ 20.000] print: 0
